data-es7.js

```diff
@@ -337,6 +337,23 @@ exports.tests = [
         babel:       true,
       }
     },
+    'await support' : {
+      exec: function () {/*
+        return (async function(){
+          return 10 + await Promise.resolve(10);
+        })() instanceof Promise
+      */},
+      res: {
+      }
+    }, 
+    'await * support' : {
+      exec: function () {/*
+      return (async function(){
+        var g = await * Promise.all([Promise.resolve(10), Promise.resolve(1)]);
```

Comment:
At the time when `await*` was part of the proposal, it would have made `Promise.all` implicit - so this isn't correct at any point in the proposal.